File: advanced/1027/1027.cpp
#include <bits/stdc++.h>
using namespace std;
char op[15] = {"0123456789ABC"};
void solve(int x)
{
    string s;
    while (x >= 13)
    {
        s = op[x%13] + s;
        x /= 13;
    }
    s = op[x] + s;
    while (s.size() < 2)
        s = op[0] + s;
    cout << s;
}
int main()
{
    int a;
    printf("#");
    for (int i = 0; i < 3; i++)
    {
        scanf("%d", &a);
        solve(a);
    }
    puts("");
	return 0;
}

File: advanced/1027/1027.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-
def solve(x):
    s, ans = '0123456789ABC', ''
    while x >= 13:
        ans = s[x%13] + ans
        x = x // 13
    ans = s[x] + ans
    while len(ans) < 2:
        ans = '0' + ans
    print(ans, end = '')
print('#', end = '')
for i in input().split():
    solve(int(i))
print('')